systemtest/cmd/sendotlp/main.go

```diff
@@ -214,9 +193,10 @@ func generateSpans(ctx context.Context, tracer trace.Tracer) error {
 }
 
 func generateMetrics(ctx context.Context, meter metric.Meter) error {
-	metric.Must(meter).NewFloat64Counter("float64_counter").Add(ctx, 1)
+	counter, _ := meter.SyncFloat64().Counter("float64_counter")
```

Comment:
Should we handle the error here? I think error may not happen while only using the command but the command may fail without a good explanation in case some error happens while developing something on top of this.

Reply:
I figured the program was small enough that it would be easy to debug, but it doesn't hurt to check - updated.